Xamarin.Forms.Maps/GeographyUtils.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Xamarin.Forms.Maps
+{
+	public static class GeographyUtils
```

Comment:
Does this *have* to be public? I'd hate for Forms to be the source of truth for how big the earth is. 😂